actions/lint-files/action.yml

```diff
@@ -0,0 +1,14 @@
+name: 'Lint files'
+description: 'Lint files'
+
+runs:
+  using: "composite"
+  steps:
+    - name: Lint script files
```

Comment:
I am not sure on the name 'Lint script files', even 'Lint files' seems a little much, as what do we lint other than files? So maybe just 'Lint'?
**OPTIONAL**

Reply:
Done